### ISSUE TITLE
ember-basic-dropdown 1.0.5 and selectSearch

### ISSUE THREAD

**Issue by nfc036** [cibernox/ember-power-select#1189]
ember-basic-dropdown 1.0.5 does show the aria-owns attribute only when dropdown is open, see this PR: https://github.com/cibernox/ember-basic-dropdown/pull/436

ember-power-select test helpers (i.e. selectSearch) are using this attribute to find the correct "contentId", see this line: https://github.com/cibernox/ember-power-select/blob/3a868463e9ff96a5ca0a89783b5e74bcac65b5b3/addon-test-support/index.js#L80

Due to this change in ember-basic-dropdown my tests are failing. Calling selectSearch() without calling "selectTrigger" or some other function to open the dropdown gives "Cannot read property 'value' of undefined" as "aria-owns" is not set.

ember-power-select@2.2.0

When using ember-basic-dropdown@1.0.4 everything is working as expected.

**Comment by cibernox** [comment]
I released 1.0.6 that fixes the problem. I'll investigate how to update `selectSearch` before updating EBD again.

**Comment by cibernox** [comment]
Fix to EPS can work with any version of EBD: https://github.com/cibernox/ember-power-select/pull/1190

**Comment by nfc036** [comment]
Thank you for that very quick fix! I can confirm, that eps@2.2.1 with ebd@1.0.6 is working as expected.

**Comment by cibernox** [comment]
I now have to revert the revert in this repo

**Comment by nfc036** [comment]
??? Do you mean going back to ebd@1.0.5 as eps@2.2.1 correctly handles the missing aria-owns!?

**Comment by cibernox** [comment]
Yes, but bumbling a bigger version number
